src/index.ts

```diff
@@ -15,51 +15,42 @@
 /**
  * HandleRequest is the entrypoint which will be invoked by the Agent.
  */
-export type HandleRequest = (request: DsrRequest, connectionConfig: ConnectionConfig) => Promise<DsrResponse>
+export type HandleRequest = (request: DSRRequest, connectionConfig: ConnectionConfig) => Promise<DSRResponse>
 
 export type DSRVersion = 'dsr/v1'
 
 /**
  * The status of the Data Subject Request.
  */
-export enum RequestStatus {
```

Comment:
I think you need to change something in tsconfig to get enums to work rather than drop them

Reply:
```
"preserveConstEnums": true,
```
did the trick.